signing_clients/apps.py

```diff
@@ -340,12 +349,12 @@ def make_signed(self, signature, outpath=None, sigpath=None):
         sigpath = os.path.join('META-INF', sigpath)
 
         with ZipFile(self.inpath, 'r') as zin:
-            with ZipFile(outpath, 'w', zipfile.ZIP_DEFLATED) as zout:
+            with ZipFile(outpath, 'w', ZIP_DEFLATED) as zout:
                 # The PKCS7 file("foo.rsa") *MUST* be the first file in the
                 # archive to take advantage of Firefox's optimized downloading
                 # of XPIs
                 zout.writestr("%s.rsa" % sigpath, signature)
-                for f in sorted(zin.infolist()):
+                for f in zin.infolist():
```

Comment:
Well, `.infolist()` returns objects and none that are properly sortable anyway so I kicked it since it made this code behave randomly already. I don't see that the ordering is relevant for the resultung zip file but please feel free to correct me if I misunderstand that.